src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -115,30 +114,30 @@ async def set_output_position(self, position_sw):
             [0; 65535] depending on the unit. This corresponds to 0 to 100% of
             the maximum piezo extension.
         """
-        payload = st.pack("<HH", 1, position_sw)
+        payload = st.pack("<HH", _CHANNEL, position_sw)
         await self.send(Message(MGMSG.PZ_SET_OUTPUTPOS, data=payload))
 
-    async def get_output_position(self):
+    async def get_output_position(self) -> int:
         """Get output position of piezo actuator.
 
         :return: The output position of the piezo relative to the zero
             position.
         :rtype: int
         """
         get_msg = await self.send_request(
-            MGMSG.PZ_REQ_OUTPUTPOS, [MGMSG.PZ_GET_OUTPUTPOS], 1
+            MGMSG.PZ_REQ_OUTPUTPOS, [MGMSG.PZ_GET_OUTPUTPOS], _CHANNEL
```

Comment:
```suggestion
            MGMSG.PZ_REQ_OUTPUTPOS, [MGMSG.PZ_GET_OUTPUTPOS], Tpz._CHANNEL
```
Use class constants.